src/js/timingDiagram.js

```diff
@@ -61,7 +61,7 @@ function timingDiagram(element, response, workflowName, height) {
             var firstEventStart = null;
             var finalEventEnd = null;
 
-            if(callStatus == "Done" || callStatus == "Failed" || callStatus == "RetryableFailure") {
+            if(callStatus == "Done" || callStatus == "Failed" || callStatus == "RetryableFailure" || callStatus == "Aborted" || callStatus == "Bypassed") {
```

Comment:
Accounted for all "terminal" statuses here: https://github.com/broadinstitute/cromwell/blob/develop/core/src/main/scala/cromwell/core/ExecutionStatus.scala#L5